Assets/_BForBoss/_Core/Scripts/UserInterface/UseCases/ResetGameUseCase.cs

```diff
@@ -0,0 +1,17 @@
+namespace BForBoss
+{
+    public class ResetGameUseCase
```

Comment:
If both ResetGame and QuitGame have the same `Execute` Method, maybe it would be nice to have an interface (i.e. `IUIStateBehaviour` or `IUIStateUseCase`) or something like that?

Reply:
This is just a naming convention thing.